src/Raven.Client/Documents/Commands/Batches/BatchCommand.cs

```diff
@@ -183,6 +183,12 @@ internal static void AppendOptions(StringBuilder sb, IndexBatchOptions indexOpti
                     }
                 }
             }
+
+            if (shardedOptions != null)
+            {
+                if (shardedOptions.BatchBehavior != ShardedBatchBehavior.SingleBucket)
```

Comment:
Isn't that the other way around? `SingleBucket` is when we error, and that will not be the default.